chainer/functions/math/clip.py

```diff
@@ -10,8 +10,11 @@ class Clip(function_node.FunctionNode):
     """Clips (limits) elements of input variable."""
 
     def __init__(self, x_min, x_max):
+        if (x_min is None) and (x_max is None):
```

Comment:
Thanks for adding the check. It's a detail but you can skip the parenthesis.
```suggestion
        if x_min is None and x_max is None:
```